packages/workbox-routing/src/_types.ts

```diff
@@ -6,25 +6,8 @@
   https://opensource.org/licenses/MIT.
 */
 
-import {
-  RouteHandler,
-  RouteHandlerObject,
-  RouteHandlerCallback,
-  RouteHandlerCallbackOptions,
-  RouteMatchCallback,
-  RouteMatchCallbackOptions
-} from 'workbox-core/types.js';
 import './_version.js';
 
- export {
-  RouteHandler as Handler,
-  RouteHandlerObject as HandlerObject,
-  RouteHandlerCallback as HandlerCallback,
-  RouteHandlerCallbackOptions as HandlerCallbackOptions,
-  RouteMatchCallback as MatchCallback,
-  RouteMatchCallbackOptions as MatchCallbackOptions,
-}
-
 // * * * IMPORTANT! * * *
 // ------------------------------------------------------------------------- //
 // jdsoc type definitions cannot be declared above TypeScript definitions or
```

Comment:
This was a problem before, but it looks like the jsdoc comments in this file are no longer up-to-date. Can you fix that as part of this PR?

Reply:
Once the actual types are 👍 , I'll go back and adjust the JSDocs. It's going to be too much wasted effort otherwise.

Reply:
SGTM